src/dpw.jl

```diff
@@ -28,13 +28,8 @@ function POMDPModelTools.action_info(p::DPWPlanner, s; tree_in_info=false)
 
         S = statetype(p.mdp)
         A = actiontype(p.mdp)
-        if p.solver.keep_tree
-            if p.tree == nothing
-                tree = DPWTree{S,A}(p.solver.n_iterations)
-                p.tree = tree
-            else
-                tree = p.tree
-            end
+        if p.solver.keep_tree && p.tree != nothing
```

Comment:
This change seems like it is probably OK so we can keep it this time, but changes that are not related to the main subject of the PR slow down reviews, so they should usually be avoided or submitted in other PRs

Reply:
Thanks, couldn't resist packing small changes into the same PR but will avoid it in future PRs :-)